packages/expo-modules-core/android/src/main/cpp/JSIInteropModuleRegistry.h

```diff
@@ -27,12 +30,25 @@ class JSIInteropModuleRegistry : public jni::HybridClass<JSIInteropModuleRegistr
 
   static void registerNatives();
 
+  /**
+   * Initializes the `ExpoModulesHostObject` and adds it to the globa object.
+   *
+   * @param jsRuntimePointer
+   * @param jsInvokerHolder
+   * @param nativeInvokerHolder
```

Comment:
I don't think we need to list the params if they don't have any description. I'm ok if they don't, it's mostly self-explanatory based on the function's description.